README.md

```diff
@@ -0,0 +1,34 @@
+# Ecoverse NFT Collection
+
+NFT Collection built with Metaplex's Candy Machine
+
+## Requirements
+
+1. NodeJS version 16 and up
+2. npm (preferred)
+3. Typescript
+
+## Local Development
+
+1. Clone the project
+2. Install
+
+## Environment variables
+
+1. Create `.env`
+
+2. Include the following environment variables
+
+   ```
+   # Set to production when deploying to production
+   NODE_ENV=development
+   # Node.js server configuration
+   SERVER_PORT=3000
+   # Quicknode RPC
+   QUICKNODE_RPC=QUICKNIDE_URL
+   # nft.storage key
+   NFT_STORAGE_KEY=NFT_STORAGE_KEY
+   # NFT Collection and Candy Machine ID
+   COLLECTION_NFT_MINT=COLLECTION_ID
+   CANDY_MACHINE_ID=CANDY_MACHINE_ID
+   ```
```

Comment:
Are some of the required env var missing?

- &rarr; add `ADMIN_KEY=...` (You used that variable in all files in the `pre-steps/` directory.)
- &rarr; add `MINTER_KEY=` (You used this variable in `frontend/minting.ts`.)